<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configDialog</class>
 <widget class="QDialog" name="configDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_privPwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;隐私密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_privPwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_secuLevel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>安全等级：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_secuLevel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_authProtocol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>认证协议：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_authProtocol">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_privProtocol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>隐私协议：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_privProtocol">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_authPwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>认证密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_authPwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_secuName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>安全名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_secuName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_contextName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>上下文名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contextName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_contextEngineID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上下文引擎ID：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contentEngineID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>70</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_communityName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_communityName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>社区名：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_privPassword">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_authPassword">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
